static/js/components/PersonalTab.js

```diff
@@ -25,26 +17,22 @@ class PersonalTab extends React.Component {
       value: language.alpha2,
       label: language.English
     }));
-    this.countryOptions = COUNTRIES.map(country => ({
-      value: country.Code,
-      label: country.Name
-    }));
     this.saveAndContinue = saveAndContinue.bind(this, '/dashboard');
```

Comment:
this we bind with a specific argument, so we still need to bind it in each tab's constructor.

we could instead make `saveAndContinue` push a `nextTabUrl` prop onto the history, and then it could be bound in `ProfileTab` instead of here, but eh, same amount of code either way (still would have to declare what the `nextTabUrl` prop looked like on every tab component).